probe/probe.go

```diff
@@ -360,119 +379,195 @@ type KubeArmorProbeData struct {
 	HostVisibility          string
 }
 
-func isKubeArmorRunning(c *k8s.Client, o Options) bool {
-	_, err := getKubeArmorDaemonset(c, o)
-	return err == nil
+func isKubeArmorRunning(c *k8s.Client, o Options) (bool, *Status) {
+	_, DaemonsetStatus, err := getKubeArmorDaemonset(c, o)
+	return err == nil, DaemonsetStatus
```

Comment:
I don't see the value of sending the error back if we are not handling it. 
```suggestion
	isRunning, DaemonsetStatus := getKubeArmorDaemonset(c, o)
	return isRunning, DaemonsetStatus
```